vllm/v1/worker/block_table.py

```diff
@@ -140,6 +204,25 @@ def clear(self) -> None:
         self.block_table.fill_(0)
         self.block_table_cpu.fill_(0)
 
+    def _convert_physical_to_logical_blocks(
+            self, physical_blocks: np.ndarray) -> np.ndarray:
+        """Convert physical block IDs to logical block IDs."""
+        if not self.use_hybrid_blocks:
+            return physical_blocks
+
+        # Create logical block IDs by splitting each physical block
+        logical_blocks: list[int] = []
```

Comment:
Can you vectorize this to avoid python loop?
Maybe:
```
bias = np.arange(1, self.blocks_per_phys_block + 1).reshape(1, -1)
logical_blocks = phsical_blocks * self.blocks_per_phys_block + bias
return logical_blocks
```